crafting-room-frontend/src/lib/server/utils/strapi.ts

```diff
@@ -0,0 +1,39 @@
+import { StrapiClient, StrapiClientOptions } from '@/lib/server/api/strapi-client';
+import { throwExp } from '@/lib/utils';
+
+const options: StrapiClientOptions = {
+	hostname: process.env.STRAPI_HOST
+		? process.env.STRAPI_HOST
+		: throwExp('Missing `STRAPI_HOST` environment variable'),
+
+	fallbackImageURL: process.env.FALLBACK_IMAGE_URL
+		? process.env.FALLBACK_IMAGE_URL
+		: '/fallback.png',
+
+	baseEndpoint: '/api'
+};
+
+if (process.env.STRAPI_MEDIA_PROVIDER_HOST)
```

Comment:
Given this site is only ever going to be deployed in one way, I'm not sure the extra complexity introduced by these new options is worth it. Do you have a use case for all of these?

Reply:
yeah i agree, its pointless. just made it easy for development honestly

they are intended to configure a dev environment which is ideal, but i suppose making them environment variables is a bit unecessary

ill remove these options from the environment variables for you 👍